.github/workflows/publish-docker-images.yml

```diff
@@ -1,55 +1,40 @@
 name: Publish Docker Images
 
 on:
-  workflow_dispatch:
+  workflow_call:
```

Comment:
this workflow is not called as part of the pre-release workflow instead of being triggered by a stable release; this allows for separation of pre-release and stable release